Service/Order/ProcessSettledWebhook.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * Copyright © TrueLayer Ltd. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace TrueLayer\Connect\Service\Order;
+
+use Exception;
+use Magento\Sales\Api\Data\OrderInterface;
+use Magento\Sales\Api\OrderRepositoryInterface;
+use Magento\Sales\Model\Order;
+use Magento\Sales\Model\Order\Email\Sender\InvoiceSender;
+use Magento\Sales\Model\Order\Email\Sender\OrderSender;
+use TrueLayer\Connect\Api\Config\RepositoryInterface as ConfigRepository;
+use TrueLayer\Connect\Api\Log\RepositoryInterface as LogRepository;
+use TrueLayer\Connect\Api\Transaction\RepositoryInterface as TransactionRepository;
+use TrueLayer\Connect\Api\User\RepositoryInterface as UserRepository;
+
+/**
+ * Class ProcessSettledWebhook
+ */
+class ProcessSettledWebhook
```

Comment:
The settled webhook does not create an order anymore, it simply updates the order status to PROCESSING.